### ISSUE TITLE
BUG - 2 - iOS/Android - Accessibility: Errors generated when submitting forms do not indicate the error has occurred

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#10841]
## What happened?

When the user is submitting a form and generates an error, the screen reader does not indicate that an error has occurred or focuses on the objects on the screen that contain the error. This can cause an accessibility user a block in progress when submitting forms, as they will hit the 'Submit' button but will remain stuck on the page until the unknown errors are resolved.

## Specs:

- Device: iOS, Android
- OS: Latest
- User Account: DEMO
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

1. Ensure the screen reader is ON, then log in to the VA app as a DEMO user.
2. Go to the Claims page, then select a claim with requests to upload files. (Compensation claim, "More information needed" tag,  January 1, 2021)
3. Review the file requests, then select a document to submit a file for.
4. Go through the upload file/photo flow.
5. On the Upload file/photo page. Leave the form in a state that will throw an error when submitting the file/photo (Deleting the file to be submitted, leaving the _Document type_ field blank, or leaving the checkbox _unchecked_.
6. Press the Submit file button and observe - The error states will display on the form, but the screen reader will **not** announce the errors are now on the page. The user's focus will still remain on the Submit file button.

## Desired behavior

The screen reader should announce that an error has occurred. The screen reader focus should move to the top-most error that occurred on the page.

## Acceptance Criteria

When a user causes an error on the page, such as when submitting a form, the screen reader should announce that an error has occurred. The screen reader should then focus on the top-most error that occurred on the page.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: **High** Low 
- Frequency: High **Low** 

- 3 - Low
- 2 - **High**
- 1 - Critical

## Screen shot(s) and additional information

[](url)

https://github.com/user-attachments/assets/217b664b-3120-4ae5-80dc-0659fbe0bf80

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by timwright12** [comment]
@jmarchi @dumathane @AdryienH @dftony @dftony @keli13 please drop an estimate here as a comment when you can

**Comment by dftony** [comment]
It looks like some sort of accessibility refocus has already been implemented under `utils/hooks/index`, and if it does what it sounds like then probably a 2-3

**Comment by jmarchi** [comment]
This error text is part of the FormWrapper system, so whatever the fix is will likely impact all forms unless there is a specific reason it's not working on this screen that we can work around. I would say a 5 with a potential that this is an easy fix, or could be something much larger that needs a wider discussion around form errors.

**Comment by dftony** [comment]
I briefly tried using the hooks I mentioned in my previous comment and it wasn't as straightforward as I anticipated, bumping to a 5 as well.

**Comment by keli13** [comment]
would give this a 5 as well. `FormWrapper` component makes this complicated even with the usage of `useAccessibilityFocus`. There is also `HeaderBanner` component would be a good place to look how that hook is being used that might help on making it work.

**Comment by digitalken** [comment]
OK on iOS and Android.

<img width="1022" height="688" alt="Image" src="https://github.com/user-attachments/assets/a7307383-e98d-469f-a114-d09ce829a68f" />
